operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/controller/UpdatableResourceCache.java

```diff
@@ -0,0 +1,10 @@
+package io.javaoperatorsdk.operator.processing.event.source.controller;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.javaoperatorsdk.operator.processing.event.ResourceID;
+
+public interface UpdatableResourceCache<T extends HasMetadata> extends ResourceCache<T> {
```

Comment:
This is not used anywhere. Where it will be?

Reply:
Ah, that shouldn't have been committed! 😅